app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertProductAttributeByCodeOnProductFormActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertProductAttributeByCodeOnProductFormActionGroup">
```

Comment:
Please, rename the action group as the following example 
```suggestion
    <actionGroup name="AssertAdminProductAttributeByCodeOnProductFormActionGroup">
```

Reply:
Renamed, thanks.